src/main/java/org/folio/rest/core/exceptions/ErrorCodes.java

```diff
@@ -104,7 +104,8 @@ public enum ErrorCodes {
   NOT_FOUND("notFound", "Not Found"),
   FORBIDDEN_DELETE_SYSTEM_VALUE("forbiddenDeleteSystemValues", "It is forbidden to delete system values"),
   FORBIDDEN_DELETE_USED_VALUE("forbiddenDeleteUsedValue", "Deleting the value used is prohibited"),
-  ERROR_REMOVING_INVOICE_LINE_ENCUMBRANCES("errorRemovingInvoiceLineEncumbrances", "Error removing invoice line encumbrance links after deleting the encumbrances: %s");
+  ERROR_REMOVING_INVOICE_LINE_ENCUMBRANCES("errorRemovingInvoiceLineEncumbrances", "Error removing invoice line encumbrance links after deleting the encumbrances: %s"),
+  ERROR_UPDATING_PO_LINE("errorUpdatingPOL", "Composite POL related invoice lines contains lines which has status APPROVED for the current fiscal year, invoice line ids: %s");
```

Comment:
This name of error code is too generic, we need specific name to show user that related invoice lines contains lines which has status APPROVED for the current fiscal year.
PO_LINE_HAS_RELATED_APPRVOVED_INVOICE_ERROR("poLineHasRelatedApprovedInvoice");
Also please agree this name with @usavkov-epam

Reply:
Name of error code updated